controllers/MoviesController.js

```diff
@@ -0,0 +1,70 @@
+const MovieModel = require('../models/MovieModel');
+const moviesController = {};
+
+
+moviesController.list = (request, response, next) => {
+  MovieModel.find().exec()
+  .then(movie => {
+    return response.json(movie);
+  })
+  .catch(err => {
+    return next(err);
+  });
```

Comment:
When using promises I would always indent the handler functions `.then`and `.catch`